components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx

```diff
@@ -107,12 +129,40 @@ const BillingSubscriptions: React.FC<Props> = () => {
         </Tooltip>
     );
 
-    const [showDanger, setShowDanger] = useState(true);
-    const [showWarning, setShowWarning] = useState(true);
-    const [showInfo, setShowInfo] = useState(true);
-
+    const [showDanger, setShowDanger] = useState(false);
+    const [showWarning, setShowWarning] = useState(false);
     const [dropdownValue, setDropdownValue] = useState<{label: string, value: string} | undefined>(undefined);
 
+    useEffect(() => {
+        if (!props.analytics) {
+            (async function getStandardAnalytics() {
+                await props.actions.getStandardAnalytics();
+            }());
+        }
+    }, []);
+
+    const shouldShowInfoBanner = (): boolean => {
+        const {analytics, userLimit, isCloud, preferences} = props;
+        if (!analytics || !isCloud || !userLimit || preferences.some((pref) => pref.name === CloudBanners.HIDE && pref.value === 'true')) {
```

Comment:
do we even need this check? since by default this function returns `false`. maybe this only needs the check that returns `true` and return `false` for everything else? would simplify the code here.

Reply:
Things like isCloud are probably not entirely necessary here, but the checks like !analytics, !userLimit, etc are needed because at this point in time they might not be loaded (see useEffect). I left the rest in because I think it's helpful to immediately see explicitly which reasons to not show the banner.